internal/server/caseload.go

```diff
@@ -48,25 +48,3 @@ func (cv CaseloadVars) buildUrl(team string) string {
 func (cv CaseloadVars) GetTeamUrl(team sirius.Team) string {
 	return cv.buildUrl(team.Selector)
 }
-
-func (cv CaseloadVars) GetReportDueDate(order []sirius.Order) string {
-	return order[0].LatestAnnualReport.DueDate
-}
-
-func (cv CaseloadVars) GetClientStatus(orders []sirius.Order) string {
-	orderStatuses := make(map[string]string)
-
-	for _, s := range orders {
-		label := s.Status.Label
-		orderStatuses[label] = label
-	}
-
-	statuses := []string{"Active", "Open", "Closed", "Duplicate"}
-	for _, status := range statuses {
```

Comment:
Could you explain what you did with the statuses. I don't quite follow what it was doing before but it looks like maybe it was ranking the orders based on status and only returning one of them.